diagnostics-app/.browserslistrc

```diff
@@ -0,0 +1,17 @@
+# This file is used by the build system to adjust CSS and JS output to support the specified browsers below.
```

Comment:
Do we really need this file as part of the repo?

Reply:
It was an autogenerated file, removed it. 👍🏻